### DISCUSSION TITLE
Backspace Key Issue in NVDA with Nudi Kannada Software

### DISCUSSION THREAD

**Discussion by VileshBandekar** [nvaccess/nvda#16284] [General]
Dear NVDA Developers,
I'm writing to report a problem I'm experiencing with NVDA while using Nudi software for typing in Kannada. When I press the backspace key, I receive a message stating:
71 hotkeys have been received in the last 344ms.
Do you want to continue?
(see #MaxHotkeysPerInterval in the help file)
This message only appears when I'm using NVDA with Nudi. It doesn't show up when I type without NVDA.
I previously reported this issue and was advised to disable "handle keys from other applications" in the settings. However, this solution doesn't work – the backspace and insert keys malfunction after disabling this option.
I would greatly appreciate it if you could investigate this issue further and find a solution that allows NVDA to coexist smoothly with Nudi for Kannada typing.
Additional Information:
• NVDA version: (NVDA Recent version)
• Nudi software version: ([nudi 6.1](https://nudityping.com/nudi-6-1-software-free-download/))
Thank you for your time and attention to this matter.
Sincerely,
Vilesh Bandekar

**Comment by XLTechie** [comment]
Why not open an actual bug issue for this? Go to 
https://github.com/nvaccess/nvda/issues, select "New issue", and use the "Get 
started" link under bug report.

Then, fill out the template that appears in the text field, which includes 
instructions.

If you do, be sure to report actual version numbers, not "recent version", 
which is not helpful for diagnostic purposes.

**Comment by lukaszgo1** [comment]
Before reporting this as an issue, please verify that this still occurs with latest Alpha version of NVDA. Alphas can be downloaded from https://www.nvaccess.org/files/nvda/snapshots/.  According to the change log for 2024.2 this issue should be fixed.

**Comment by seanbudd** [comment]
This has already been fixed for 2024.2 - see #15822
